lib/cylc/loggingutil.py

```diff
@@ -106,7 +106,12 @@ def should_rollover(self, record):
         if max_bytes < self.MIN_BYTES:  # No silly value
             max_bytes = self.MIN_BYTES
         msg = "%s\n" % self.format(record)
-        self.stream.seek(0, 2)  # due to non-posix-compliant Windows feature
+        try:
+            # due to non-posix-compliant Windows feature
+            self.stream.seek(0, 2)
+        except ValueError as exc:
+            # intended to catch - ValueError: I/O operation on closed file
+            raise SystemExit(exc)
```

Comment:
I guess this is only for double safety? Surprised that this is not an `IOError` or `OSError`.

Reply:
> double safety

Yep, I don't think this is actually needed provided we ensure logging closure always gets wrapped in a `try, except, IOError` block.